### ISSUE TITLE
Contribution guidelines are inconsistent

### ISSUE THREAD

**Issue by pmjordan** [oasis-tcs/tosca-specs#289]
CSD06 has an opening section headed 'Status' which details how TC members and the public should comment on the document. The text has been copied from V1.3 and does not take account of our current use of tosca-community-contributions on github.

The CONTRIBUTING.md document of tosca-specs invites contributions to that repo and does not mention the use of tosca-community-contributions. It also points to the OASIS TC mailing list pages.

If the use of tosca-community-contributions is to continue after publication of V2.0 then that should be stated in the document and it's repo. If not then the CONTRIBUTING.md from tosca-community-contributions should be copied to tosca-specs in preparation for tracking issues on that repo.

**Comment by lauwers** [comment]
the tosca-community-contributions repo is a *community* forum only and in theory should not be used for work on the specification. I believe the text in the document correctly reflects OASIS's policies. However, OASIS is currently also reviewing these policies, and we should encourage the use of github rather than email for commenting on the spec. In that case, we should use https://github.com/oasis-tcs/tosca-specs/issues rather than this repository.

**Comment by pmjordan** [comment]
Then let's amend the CSD to point to https://github.com/oasis-tcs/tosca-specs/issues as the primary feedback channel but that comments may also be submitted using the prevailing OASIS method.

**Comment by lauwers** [comment]
It appears that the new template that was used for the Markdown version of the spec no longer includes that language that was in CSD06. Instead, the github repository has a separate file (in https://github.com/oasis-tcs/tosca-specs/blob/working/CONTRIBUTING.md) that spells out where feedback should be submitted. It does point to the github issues mechanism as the primary channel.

**Comment by pmjordan** [comment]
Yes I saw and mentioned https://github.com/oasis-tcs/tosca-specs/blob/working/CONTRIBUTING.md in my original issue report. I agree that that will be the route that people would expect to use for comments on version 2 once it has been published and I agree that is the route which we should promote.
However I still think there is an inconsistency; line 55 Instructs users to submit comments via the OASIS mailing list. I think that line should be either be removed or revised to point at CONTRIBUTING.md
